cf/data/data.py

```diff
@@ -9219,7 +9223,8 @@ def exp(self, inplace=False, i=False):
         if d.Units:
```

Comment:
Could we f-string-ify the above error message?

Reply:
We can indeed. I will add that in a new commit in a moment...